### PR TITLE
Use standard JspBase.getWebappURL() and JspBase.getScriptTag() helper methods

### PR DESCRIPTION
**Pull request by labkey-adam**
#### Rationale
We have helper methods that safely and succinctly create `<script>` and `` tags.

#### Related Pull Requests
* https://github.com/LabKey/platform/pull/1649

#### Changes
* Use standard JspBase.getWebappURL() and JspBase.getScriptTag() helper methods instead of hand assembling script include tags and image URLs

### CODE REVIEW

File: src/org/labkey/cds/view/template/ConnectorTemplate.jsp [hunk 1]
```diff
@@ -97,282 +94,292 @@
...
+
+            srcPath + "/Application.js",
+
+            srcPath + "/app.js"
+        ));
+    }
+    else
+    {
+        // PRODUCTION
+        includePaths.add("/Connector/extapp.min.js?v=" + serverHash);
```
Comment (labkey-matthewb) [inline src/org/labkey/cds/view/template/ConnectorTemplate.jsp]: Don't add serverHash

Comment (labkey-adam) [reply src/org/labkey/cds/view/template/ConnectorTemplate.jsp]: Fixed

File: src/org/labkey/cds/view/template/ConnectorTemplate.jsp [hunk 2]
```diff
@@ -52,11 +48,11 @@
...
     <title>DataSpace</title>
 
-    <link rel="icon" type="image/png" href="<%=text(appPath)%>/images/headerlogo.png">
+    <link rel="icon" type="image/png" href="<%=getWebappURL("/Connector/images/headerlogo.png")%>">
 
     <!-- stylesheets -->
-    <link type="text/css" href="<%=text(contextPath)%>/hopscotch/css/hopscotch.min.css" rel="stylesheet">
-    <link type="text/css" href="<%=text(resourcePath)%>/Connector-all.css<%= text(devMode ? "" : ("?"+serverHash)) %>" rel="stylesheet">
+    <link type="text/css" href="<%=getWebappURL("/hopscotch/css/hopscotch.min.css")%>" rel="stylesheet">
+    <link type="text/css" href="<%=getWebappURL(resourcePath + "/Connector-all.css" + (devMode ? "" : "?" + serverHash))%>" rel="stylesheet">
```
Comment (labkey-matthewb) [inline src/org/labkey/cds/view/template/ConnectorTemplate.jsp]: unnecessary serverHash?

**Review by labkey-matthewb — APPROVED** [review]
I think there's one more serverHash to fix, but otherwise yeah.

**Pull request merged**